test/unit_Polytope.jl

```diff
@@ -56,6 +56,13 @@ for N in [Float64, Rational{Int}, Float32]
         @test length(p.constraints) == length(cl)
     end
 
+    # convert a hyperrectangle to a HPolytope
+    H = Hyperrectangle(N[1, 1], N[2, 2])
+    P = convert(HPolytope, H1)
+    vlist = vertices_list(P)
+    @test length(vlist) == 4
+    @test all([vi ∈ vlist for vi in [N[3, 3], N[3, -1], N[-1, -1], N[-1, 3]]]) 
```

Comment:
i added this test here because the conversion and testing for vertices requires to load the package `Polyhedra`

Reply:
You have to add a condition that `Polyhedra` is loaded around this test. Otherwise, `vertices_list` is not defined and the test crashes.